tests/commands/test_version_command.py

```diff
@@ -185,6 +185,56 @@ def test_version_just_major_error_no_project(config, capsys, argument: str):
     captured = capsys.readouterr()
     assert not captured.out
     assert (
-        "Major or minor version can only be used with --project or --verbose."
-        in captured.err
+        "can only be used with MANUAL_VERSION, --project or --verbose." in captured.err
     )
+
+
+@pytest.mark.parametrize(
+    "next_version, current_version, expected_version",
```

Comment:
maybe rename `next_version` to `next_increment`?
In the user facing cli, it makes sense semantically: `--next=MAJOR` (give me the next major).